CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ endif()
 
 project(OpenEXRMetaProject)
 
+# Search for custom modules in our cmake directory
+list (APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
```

Comment:
I don't know why, but I had been avoiding changing the module path, and just including cmake/foo. I could have sworn I saw somewhere that to support other projects including your project as a sub-project, you shouldn't adjust the module path. But now I can't find that reference. Either, way, is there a particular reason to add the path vs. just including directly?

Reply:
Oh, sure. I was just trying to keep the include directives clean, but if you'd rather they be explicit, that's fine.

Reply:
pushed an update to address this